# Patches for microb-engine
# Oleg Romashin <ext-oleg.2.romashin@nokia.com>
# DP: HACK Disable showing some unicode symbols

Index: layout/generic/nsTextTransformer.h
===================================================================
--- layout/generic/nsTextTransformer.h.orig
+++ layout/generic/nsTextTransformer.h
@@ -81,17 +81,17 @@
 #endif // IBMBIDI
 
 // For now, we have only a couple of characters to strip out. If we get
 // any more, change this to use a bitset to lookup into.
 //   CH_SHY - soft hyphen (discretionary hyphen)
 #ifdef IBMBIDI
 // added BIDI formatting codes
 #define IS_DISCARDED(_ch) \
-  (((_ch) == CH_SHY) || ((_ch) == '\r') || IS_BIDI_CONTROL(_ch))
+  (((_ch) == CH_SHY) || ((_ch) == '\r') || ((_ch) == 0xfffd) || IS_BIDI_CONTROL(_ch))
 #else
 #define IS_DISCARDED(_ch) \
   (((_ch) == CH_SHY) || ((_ch) == '\r'))
 #endif
 
 /* Unicode codepoints that could be the first of a lam alef pair converted to
  * a lam alef ligature by ArabicShaping()
  */
